package.json

```diff
@@ -2,27 +2,32 @@
   "main": "node_modules/expo/AppEntry.js",
   "private": true,
   "dependencies": {
-    "@firebase/firestore": "~0.3.4",
     "axios": "~0.17.1",
-    "expo": "^28.0.0",
-    "firebase": "~5.0.4",
+    "expo": "^36.0.0",
+    "expo-facebook": "~8.0.0",
+    "expo-image-picker": "~8.0.1",
+    "expo-linear-gradient": "~8.0.0",
+    "expo-location": "~8.0.0",
+    "expo-permissions": "~8.0.0",
+    "expo-web-browser": "~8.0.0",
```

Comment:
For expo-specific library imports, we need to now import them from the specific expo library instead of just `import {whatever} from expo`. To support this, we needed to also npm install those specific libraries.